runtests.py

```diff
@@ -2,18 +2,44 @@
 # -*- coding: utf-8 -*-
 
 import os
+import shutil
 import sys
 
-sys.path.insert(0, 'tests')
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp2.settings')
+from django.core.management import execute_from_command_line
+
+
+def check_geckodriver():
```

Comment:
Why don't we do this in the selenium mixin instead of here?
That why we get this done automatically for all selenium tests.